### PR TITLE
add base email template, add account verified email

### PR DESCRIPTION
**Pull request by jake-kent**

### CODE REVIEW

File: Maco/accounts/views.py [hunk 1]
```diff
@@ -134,11 +137,24 @@ class VerificationView(TemplateView):
     def get(self, *args, **kwargs):
         try:
             self.verify(*args, **kwargs)
-            return HttpResponseRedirect(self.request.user.get_wishlist_url())
         except VerificationError as e:
             messages.error(self.request, e.args[0])
         else:
-            messages.success(self.request, 'You have successfully verified your account')
+            messages.info(self.request, 'You have successfully verified your account')
+            domain = 'https://{}'.format(Site.objects.first().domain)
```
Comment (gone) [inline Maco/accounts/views.py]: You've done this enough that you should probably make a utli func

File: Maco/checkout/views.py [hunk 2]
```diff
@@ -149,6 +149,14 @@ def handle_successful_order(self, order):
             success_url = '{}?fulfillment_order=true'.format(success_url)
         else:
             order.send_contribution_notification_email()
+            fully_funded_lines = set()
+            for line in order.lines.all():
+                if line.basket_line and line.basket_line.wishlist_line\
```
Comment (gone) [inline Maco/checkout/views.py]: Move this check to the line object

File: Maco/accounts/views.py (hunk 1 above)
Comment (jake-kent) [reply Maco/accounts/views.py]: okay, move over to util function

File: Maco/checkout/views.py (hunk 2 above)
Comment (jake-kent) [reply Maco/checkout/views.py]: okay, moved to order line

**Pull request merged**